### PR TITLE
Feature/request instrumentation

### PR DESCRIPTION
**Pull request by ocastx**
Instrumentation implementation for https://www.npmjs.com/package/request

### CODE REVIEW

File: packages/zipkin-instrumentation-request/src/wrapRequest.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+      const headers = getHeaders(traceId, opts);
+      const zipkinOpts = Object.assign({}, opts, {headers});
+
+      request(zipkinOpts, function (error, response, body) {
+
+        if (error) {
+          tracer.scoped(() => {
+            tracer.setId(traceId);
+            tracer.recordBinary('request.error', error.toString());
```
Comment (adriancole) [inline packages/zipkin-instrumentation-request/src/wrapRequest.js]: in zipkin, "error" is the tag name that will color the span red in the UI

File: packages/zipkin-instrumentation-request/test/integrationTest.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+        const spanIdOnServer = data.spanId;
+        expect(spanIdOnServer).to.equal(spanId);
+
+        done();
+      });
+    });
+  });
+
+
```
Comment (adriancole) [inline packages/zipkin-instrumentation-request/test/integrationTest.js]: is there a way to test the "error" case?

**Comment by adriancole** [comment]
add a readme like: https://github.com/openzipkin/zipkin-js/blob/master/packages/zipkin-instrumentation-fetch/README.md ?

**Comment by adriancole** [comment]
Thanks for putting this together! I'll now defer to ppl who actually know javascript :)

**Comment by eirslett** [comment]
> I'll now defer to ppl who actually know javascript :)

You know JavaScript pretty well by now :wink:

**Pull request closed without merging**